adapter/pkg/synchronizer/apis_fetcher.go

```diff
@@ -96,6 +96,7 @@ func FetchAPIs(id *string, gwLabel *string, c chan SyncAPIResponse) {
 		if err != nil {
 			logger.LoggerSync.Errorf("Error occurred when readin the cert form %v : %v", certPath, err)
 		}
+		// TODO: (VirajSalaka) Reuse the adapter truststore configuration ?
```

Comment:
Reuse the existing truststore.

Reply:
do the same for consul certificate as well.

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1582/commits/02d61d2a72f2d3d089f049ad719122050faab739